bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,837 +4,558 @@
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Co2</label>
-		<description>Minimum CO2 on current day</description>
-		<category>Carbondioxide</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="minCo2ThisWeek" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Co2 This Week</label>
-		<description>Minimum CO2 this week</description>
-		<category>Carbondioxide</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="minCo2ThisMonth" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Co2 This Month</label>
-		<description>Minimum CO2 this month</description>
-		<category>Carbondioxide</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="maxCo2" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Max Co2</label>
-		<description>Maximum CO2 on current day</description>
-		<category>Carbondioxide</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="maxCo2ThisWeek" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Max Co2 This Week</label>
-		<description>Maximum CO2 this week</description>
-		<category>Carbondioxide</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="maxCo2ThisMonth" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Max Co2 This Month</label>
-		<description>Maximum CO2 this month</description>
-		<category>Carbondioxide</category>
+		<description>Air Quality indicator.</description>
+		<category>carbondioxide</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>CO2</tag>
+		</tags>
 		<state readOnly="true" pattern="%d %unit%"/>
 	</channel-type>
 
 	<channel-type id="noise">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Noise</label>
-		<description>Current Noise Level</description>
-		<category>Noise</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="minNoise" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Noise</label>
-		<description>Minimum Noise on current day</description>
-		<category>Noise</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="minNoiseThisWeek" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Noise This Week</label>
-		<description>Minimum Noise this week</description>
-		<category>Noise</category>
-		<state readOnly="true" pattern="%d %unit%"/>
-	</channel-type>
-
-	<channel-type id="minNoiseThisMonth" advanced="true">
-		<item-type>Number:Dimensionless</item-type>
-		<label>Min Noise This Month</label>
-		<description>Minimum Noise this month</description>
-		<category>Noise</category>
+		<description>Current Noise Level.</description>
+		<category>soundvolume</category>
```

Comment:
```suggestion
		<category>SoundVolume</category>
```

Reply:
Corrected

Reply:
Half done, to be really nitpicking, it should be Sound**V**olume. :)

Reply:
Done